<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Paient Information</string>
    </property>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>156</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>271</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>27</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>DCM Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As(S)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit(Q)</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
